api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -39,25 +41,25 @@ describe("DynamoBusinessesDataClient", () => {
     taxFilingData: generateTaxFilingData({
       filings: [],
     }),
+    version: CURRENT_VERSION,
   });
 
   beforeEach(() => {
+    logger = DummyLogWriter;
     client = DynamoDBDocumentClient.from(new DynamoDBClient(config), dynamoDbTranslateConfig);
-    dynamoBusinessesDataClient = DynamoBusinessesDataClient(client, dbConfig.tableName);
+    dynamoBusinessesDataClient = DynamoBusinessesDataClient(client, dbConfig.tableName, logger);
   });
 
   it("inserts and retrieves items from the db", async () => {
-    await expect(dynamoBusinessesDataClient.get("some-id")).rejects.toEqual(new Error("Not found"));
+    await expect(dynamoBusinessesDataClient.get(errorId)).rejects.toEqual(
+      new Error(`Business with ID ${errorId} not found in table ${dbConfig.tableName}`)
+    );
 
     const businessId = businessData.id;
     await dynamoBusinessesDataClient.put(businessData);
 
     const expectedValue = await dynamoBusinessesDataClient.get(businessId);
-    const { version, ...filteredExpectedValue } = expectedValue as unknown as {
-      version?: string;
-      [key: string]: any;
-    };
-    expect(filteredExpectedValue).toEqual(businessData);
+    expect(expectedValue).toEqual(businessData);
   });
 
   it("finds a business by the businessName", async () => {
```

Comment:
Ideally our random strings should be truly random. Ex: on line 67 we say
```js
findByBusinessName("some-business-name-59157929")
```

this should be
```js
findByBusinessName("some-business-name-${randomInt()}")
```